letters_count.py

```diff
@@ -0,0 +1,44 @@
+import string
+
+text = "Python is an interpreted high-level programming language for general-purpose programming. Created by Guido " \

```

Comment:
Python has a special syntactic for multiline strings - https://www.w3schools.com/python/gloss_python_multi_line_strings.asp. Could you please use it instead of escaping spaces?